tests/parametric/test_tracer_flare.py

```diff
@@ -115,20 +115,23 @@ def test_telemetry_app_started(self, library_env, test_agent, test_library):
         events = test_agent.wait_for_telemetry_event("app-started")
         assert len(events) > 0
 
+    @missing_feature(library="php", reason="Not implemented; https://datadoghq.atlassian.net/browse/APMLP-195")
```

Comment:
Just set the ticket id for reason (same comment for other skips)

```suggestion
    @missing_feature(library="php", reason="APMLP-195")
```